### DISCUSSION TITLE
2021-05-20

### DISCUSSION THREAD

**Discussion by jjunyjjuny** [rla36/airbnb#14] [Daily]
# 2021-05-20 하루 회고하기

**Comment by jjunyjjuny** [comment]
마크업 하는데 하루종일 걸리는듯 ㄷㄷ.. 하드코딩 보단 더 나은 구조를 고민하느라 그렇긴 하지만 그럼에도 속도가 좀 느린 것 같다

이제 typescript까지 더하면 더 느려지겠지.. 아마 다음주는 내내 라이브러리만 만들지도 모르겠다.

**Comment by rla36** [comment]
집중력이 부족한 느낌이다. jdbc Template 공부했으니까 바로 적용할 수 있을 줄 알았는데, 역시 실전은 또 다르다. 
그래도 코쿼 나가서 아이작에게 질문할 수 있어 좋았다.
